tests/airplay.test.ts

```diff
@@ -0,0 +1,94 @@
+import { describe, it, expect, beforeEach, vi } from 'vitest';
+import { VideoJsPlayer } from '../@types/videojs';
+import type { AirPlayManagerType } from '../src/js/airplay/AirPlayManager';
+import { AirPlayManager } from '../src/js/airplay/AirPlayManager';
+
+describe('AirPlayManager', () => {
+   let mockPlayer: VideoJsPlayer,
+       mockRemotePlayback: any,
+       airPlayManager: AirPlayManagerType;
```

Comment:
```suggestion
       airPlayManager: AirPlayManager;
```

Should this be able to just be type `AirPlayManager` and then delete 3?